### PR TITLE
Improved README.md

### PR DESCRIPTION
**Pull request by petervwyatt**
Based on discussion from https://github.com/veraPDF/veraPDF-arlington-tools/issues/6

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,56 @@
...
 
 This repository implements the Java-based utility performing this translation.
 
+Alington types
+==============================
+The Arlington PDF model utilizes an [expanded set of types](https://github.com/pdf-association/arlington-pdf-model/blob/master/INTERNAL_GRAMMAR.md#column-2---type) in order to more precisely define PDF objects and data integrity relationships. veraPDF fully supports this expanded set and implements the necessary additional checks. These checks include:
+- all dictionary keys must be direct objects
+- `dates` must be a valid PDF date string
+-` rectangle` must be a PDF array with precisely 4 entries
+- `matrix` must be a PDF array with precisely 6 entries and represent a valid affine transform
```
Comment (MaximPlusov) [inline README.md:16]: veraPDF doesn't check that matrix represent a valid affine transform

File: README.md [hunk 2]
```diff
@@ -1,12 +1,56 @@
...
+- `bitmask` must be a PDF integer object
+- `string-ascii` must not start with a Unicode byte order marker
```
Comment (MaximPlusov) [inline README.md:18]: For string-ascii verapdf checks only that all string characters are included in a specific set of 128 characters

File: README.md [hunk 3]
```diff
@@ -1,12 +1,56 @@
...
 
 veraPDF adds the support for this model by translating TSV files into its own validation profile based on [veraPDF formal grammar](https://docs.verapdf.org/validation/rules/) for validation rules. 
 
 This repository implements the Java-based utility performing this translation.
 
+Alington types
+==============================
+The Arlington PDF model utilizes an [expanded set of types](https://github.com/pdf-association/arlington-pdf-model/blob/master/INTERNAL_GRAMMAR.md#column-2---type) in order to more precisely define PDF objects and data integrity relationships. veraPDF fully supports this expanded set and implements the necessary additional checks. These checks include:
+- all dictionary keys must be direct objects
```
Comment (MaximPlusov) [inline README.md:13]: veraPDF doesn't check that `all dictionary keys must be direct objects`. If a dictionary key is indirect object, we will throw exception `invalid pdf dictionary`.

Comment (petervwyatt) [reply README.md:13]: I assume this gets reported so isn't this the same thing?
Or does processing terminate?

File: README.md (hunk 2 above)
Comment (petervwyatt) [reply README.md:18]: Meaning 7-bit (<= 0x7F) only?

Comment (MaximPlusov) [reply README.md:18]: Yes

File: README.md (hunk 3 above)
Comment (MaximPlusov) [reply README.md:13]: Yes, processing terminates.
image

Comment (petervwyatt) [reply README.md]: New file pushed.

File: README.md (hunk 1 above)
Comment (petervwyatt) [reply README.md]: New file pushed.

File: README.md (hunk 2 above)
Comment (petervwyatt) [reply README.md]: New file pushed.

**Pull request merged**